modules/doc/content/application_usage/failed_solves.md

```diff
@@ -101,6 +101,25 @@ Initial residual before setting preset BCs: 65444.1
 
 In this case you are likely to require many nonlinear iterations as well, but the reason is that your linear iterations don't drop. This could be due to missing terms or errors in your Jacobian or because the way you are applying your preconditioner in PETSc is not good for the problem. Make sure your Jacobian is correct and add off-diagonal terms for multivariable problems.
 
+An additional possible reason for a poor linear solve is that your problem is very poorly
+conditioned. You can inspect the condition number of your matrix for small problems (should be less
+than 1000 degrees of freedom) by running with the PETSc options `-pc_type svd
+-pc_svd_monitor`. These options will tell you your condition number as well as how many singular
+values you have. If you have any singular values, then you may have omitted a boundary condition,
+you may have a null space (all Neumann boundary conditions for example), or you may have very poor
+scaling between variables in a multi-physics simulation. You may even have run into issues if you
+have nodal boundary conditions (which introduce values of unity on the diagonals) and the Jacobian
+entries from your physics (kernels) are very large. You want your condition number to be as close to
+unity as possible. To address the latter problem or poor relative scaling between variables, you
+can use MOOSE's automatic scaling feature which will bring different physics Jacobians as close to
+unity as possible. To turn on this feature, add this block to your input file:
+
+```
+[Problem]
+  automatic_scaling = true
```

Comment:
It might be better to add this parameter to executioner  block

Reply:
I'm not sure I agree. We use the `[Problem]` block for controlling a lot of application-level parameters already - even ones you might argue fit with the executioner just as well (e.g. `solve`). Also keep in mind that we will be adding the legacy flags so that we can control this behavior on an "App" to "App" basis. I think Problem is the right place for this.

Reply:
I still like this kind of thing to Executioner. In order to use `auto scaling`, I have to create a problem block?

Reply:
Yes - but ideally users won't be doing this for an extended period of time. We'll get the legacy flag logic in there so individual apps can switch it on/off globally. Hopefully this isn't a parameter that you'll have to use very often if your application is making an effort to switch over to automatic scaling (regolding and evaluating failing tests).

Reply:
I am currently working on a commit where I've switched it over to the `Executioner`. I don't really care; it sounds like there are arguments both ways. In terms of code I think it fits much more naturally with the Problem, but for the user the Executioner seems better.